apps/studio/src/server/modules/site/site.service.ts

```diff
@@ -86,27 +86,31 @@ export const getNotification = async (siteId: number) => {
       message: "Site not found",
     })
   }
-  // NOTE: Empty string denotes absence of notification on site.
-  return result.content?.[0]?.text ?? ""
+  return result
 }
 
 interface SetSiteNotificationParams {
   tx: Transaction<DB>
   siteId: number
   userId: string
-  notification: string // TODO: Replace this with Tiptap schema once frontend refactored
+  title: string
+  content: Notification["content"]
+  enabled: boolean
```

Comment:
should we derive all these from `Notification` schema?

Reply:
done downstream!